src/partials/products.html

```diff
@@ -0,0 +1,30 @@
+<section class="products">
+  <a class="products__label" id="our_products">100% natural</a>
```

Comment:
Why is this: id="our_products" if it is not necessary please remove.

Reply:
This id is for the label, so that from the navigation you can go directly to this section. I found this particular id in "index.html".